homeassistant/components/sensor/velo.py

```diff
@@ -0,0 +1,105 @@
+"""
+Shows the available amount of public city bikes for Velo Antwerpen.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.velo/
+"""
+
+import logging
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Velo'
+DEFAULT_ICON = 'mdi:bike'
+DEFAULT_UNIT = 'bikes'
+
+CONF_STATION_ID = 'station_id'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATION_ID): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Velo sensor."""
+    name = config.get(CONF_NAME)
+    station = config.get(CONF_STATION_ID)
```

Comment:
You should check if the station is valid. If not then the setup should fails as there is no way to recover and the users wil end-up with a non-working platform.